ClassLibrary/ClassLibrary/Reader.cs

```diff
@@ -52,12 +50,12 @@ public Reader(String NameReader, String LastNameReader)
             this.LastName = LastNameReader;
         }
 
-        public void AddBookInCard(Book book)
+        internal void AddBookInCard(Book book)
         {
-            if (book == null)
-            {
-                throw new ArgumentNullException("Book's reference is Null (received in AddBookInCardReader)");
-            }
+            //if (book == null)
```

Comment:
why is this commented out?

Reply:
not necessary = I delete this
BorrowBook is exist verification, and this is 'internal'